promdens.py

```diff
@@ -689,10 +696,22 @@ def plot_pda(ics: InitialConditions) -> None:
     emin, emax = np.min(ics.spectrum[0]/ics.evtoau), np.max(ics.spectrum[0]/ics.evtoau)
     tmin, tmax = np.min(ics.field_t/ics.fstoau), np.max(ics.field_t/ics.fstoau)
 
-    h = axs.hist2d(ics.ics[3]/ics.evtoau, ics.ics[1]/ics.fstoau, range=[[emin, emax], [tmin, tmax]], bins=(100, 100), cmap=plt.cm.viridis,
-                   density=True)
-    if lchirp != 0:
-        axs.plot((omega + 2*lchirp*ics.field_t)/ics.evtoau, ics.field_t/ics.fstoau, color='white', linestyle='--', label=r"$\omega(t)$")
+    axs.hist2d(
+        ics.ics[3] / ics.evtoau,
+        ics.ics[1] / ics.fstoau,
+        range=[[emin, emax], [tmin, tmax]],
+        bins=(100, 100),
+        cmap=plt.cm.viridis,
+        density=True,
+    )
+    if ics.field_lchirp != 0:
+        axs.plot(
+            (ics.field_omega + 2 * ics.field_lchirp * ics.field_t) / ics.evtoau,
```

Comment:
Here I've switched from `omega` to `ics.field_omega` and `lchirp` to `ics.field_lchirp`.